src/elm/Pages/Build/View.elm

```diff
@@ -121,16 +123,16 @@ viewBuild { time, build, steps, logs, followingStep, shift } org repo =
                     )
 
         buildSteps =
-            case steps of
+            case model.steps of
                 RemoteData.Success steps_ ->
-                    viewSteps time org repo buildNumber steps_ logs followingStep shift
+                    viewPipeline model <| BuildModel org repo buildNumber steps_
```

Comment:
if we construct and pass in a BuildModel here, we dont need to pass along 12 arguments for each function that is deeper.